integration_tests/integration/probationPractitionerReferrals.cy.js

```diff
@@ -472,7 +472,7 @@ describe('Probation practitioner referrals dashboard', () => {
       .should('contain', 'English')
       .should('contain', 'Male')
       .should('contain', 'Agnostic')
-      .should('contain', '1 Jan 1980 (43 years old)')
+      .should('contain', '1 Jan 1980 (44 years old)')
```

Comment:
@awhodgson256 I feel its worth making this dynamic so that it won't fail the next year.

Reply:
@awhodgson256 we have used moment framework for testing the dates dynamically. we can use moment here as well